src/ui/public/agg_types/buckets/terms.js

```diff
@@ -92,9 +93,14 @@ export default function TermsAggDefinition(Private) {
           $scope.$watch('responseValueAggs', updateOrderAgg);
           $scope.$watch('agg.params.orderBy', updateOrderAgg);
 
+          $scope.rejectAgg = function (agg) {
+            // aggFilter elements all starts with a '!'
+            // so the index of agg.type.name in a filter is 1 if it is included
+            return Boolean(aggFilter.find((filter) => filter.indexOf(agg.type.name) === 1));
+          };
```

Comment:
I'm curious, why use this custom function to disable options instead of just using the existing `aggFilter` to filter them out? And if you're not using `aggFilter`, why follow its filter schema? This logic could be a lot simpler if you didn't have to account for the `!`, you could just use `_.contains`